extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/scores.html

```diff
@@ -188,8 +184,8 @@
           {/if}
           <tr>
             <td style="width:30%;">Filters</td>
-            <td style="width:70%;">
-              <span class="badge badge-light" id="collapse-list" data-toggle="collapse" data-target="#filter">
+            <td style="width:70%;" id="collapse-list" data-toggle="collapse" data-target="#filter">
+              <span class="badge badge-light">
```

Comment:
Now the complete row is clickable :) But I didn't remove the icon `>`, should I @FroMage?  I was not sure if a blank row will look nice :)